pkg/account/model.go

```diff
@@ -0,0 +1,20 @@
+package account
+
+const ACCOUNT_PASSWD_LEN = 8
+
+type Account struct {
+	Email string `gorm:"primarykey"`
+
+	// Type   AcountType
+	Type   int
```

Comment:
A human readable word like "patient" or "doctor" is preferable to me. Storage is cheap, readability is much more expensive.